admin/class-premium-upsell-admin-block.php

```diff
@@ -58,8 +58,16 @@ public function render() {
 
 		/* translators: %s expands to "Yoast SEO Premium". */
 		$dismiss_msg = sprintf( __( 'Dismiss %s upgrade motivation', 'wordpress-seo' ), 'Yoast SEO Premium' );
-		/* translators: %s expands to "Yoast SEO Premium". */
-		$upgrade_msg = sprintf( __( 'Find out why you should upgrade to %s &raquo;', 'wordpress-seo' ), 'Yoast SEO Premium' );
+
+		/* translators: %s expands to Yoast SEO Premium */
+		$button_text = sprintf( __( 'Get %s', 'wordpress-seo' ), 'Yoast SEO Premium' );
+
+		$upgrade_button = sprintf(
+			'<a id="wpseo-%1$s-popup-button" class="yoast-button-upsell" href="%2$s" target="_blank" rel="noreferrer noopener">%3$s</a>',
```

Comment:
There's a new pattern to use for links with target _blank noreferrer noopener, see https://github.com/Yoast/wordpress-seo/issues/4605#issuecomment-425409923 but I guess it should be addressed separately.